tests/osn-tests/src/test_osn_global.ts

```diff
@@ -24,23 +25,29 @@ describe('osn_global', () => {
     });
 
     context('# SetOutputSource', () => {
-        it('should set source to output channel', () => {
-            let source: IInput;
+        let source: IInput;
 
+        it('should set source to output channel', () => {
```

Comment:
general feedback: I would change `should set source to output channel` to `Set source to output channel`, clearer and we repeat less across the tests.

Reply:
Sounds good, thanks